provision/initramfs/capabilities/transport-http/functions

```diff
@@ -15,10 +15,22 @@ if [ -f "/warewulf/config" ]; then
     . /warewulf/config
 fi
 
+if [ -f "/etc/functions" ]; then
+    . /etc/functions
+fi
+
 START=0
 BACKOFF=2
 INTERVALS=30
 COUNT=1
+
+# If we want to debug, change some values
+if [! -z "$WWDEBUG" ]; then
```

Comment:
Isn’t ‘-n’ the negate of -z?

Reply:
Yeah... habbit of just doing 'not'. I'll change it to `-n`